packages/govuk-frontend/src/govuk/components/warning-text/_index.scss

```
@@ -1,13 +1,12 @@
 @include govuk-exports("govuk/component/warning-text") {
   .govuk-warning-text {
-    position: relative;
+    @include govuk-font($size: 19, $weight: bold);
```

Can we rely on `<strong>` in the template instead?

```suggestion
    @include govuk-font($size: 19);
```

---

This would mean it didn't apply to the "!" because the icon itself is secretly typography, but we can just use a font weight mixin for that. Will fix.

---

Appreciate we're squeezing the life out the CSS file size, so don't have to go this far 🙈